micromind/core.py

```diff
@@ -574,6 +594,10 @@ def train(
             else:
                 val_metrics = train_metrics.update({"val_loss": loss_epoch / (idx + 1)})
 
+            if self.hparams.log_wandb:  # wandb log
```

Comment:
I might be wrong, but it would be nicer if we divide the train and valid panes inside the wandb logging. this would look something like `val/{metric_name}` will be logged instead of `val_{metric_name}`. ofc this change only applies to the `.log` call and should not be propagated to the entire core.